### DISCUSSION TITLE
Broker started to disconnect and reject new connections due to PVC disk almost 100%

### DISCUSSION THREAD

**Discussion by Zodor** [arkmq-org/arkmq-org-broker-operator#939] [Q&A]
We had a problem last week when our Mule Java instances that connected to the Artemis cluster started to crash.
The problem was found to be the PVC'a closing in on 100% (91% actually) and the connections to the brokers was blocked.
We do not know why the PVC filled up at this moment, but we have increased the memory for them.

Any ideas what could be the problem. Should not the PVC storage just be a temporary one for incoming queue items and items be removed when processed?

**Comment by brusdev** [comment]
@Zodor old ArtemisCloud versions were saving logs in PVC storage. Could you try the latest version?
